demo_toolbox.py

```diff
@@ -26,9 +26,41 @@
     parser.add_argument("--low_mem", action="store_true", help=\
         "If True, the memory used by the synthesizer will be freed after each use. Adds large "
         "overhead but allows to save some GPU memory for lower-end GPUs.")
+    parser.add_argument("--repeatable", action="store_true", help=\
+        "If True, the synthesizer and vocoder will use hardcoded seeds for random number generation"
+        "and the models will be reloaded on each use for repeatability of voice cloning output.")
     args = parser.parse_args()
     print_args(args, parser)
 
+    ## Set seeds if repeatable output requested
```

Comment:
The code comes from here: https://stackoverflow.com/a/54048709
Should it be cleaned up so it is more concise? We can also set everything to the same seed value instead of incrementing.